airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/destination_async/DetectStreamToFlush.java

```diff
@@ -28,7 +28,7 @@
 public class DetectStreamToFlush {
 
   private static final double EAGER_FLUSH_THRESHOLD = 0.90;
-  private static final long QUEUE_FLUSH_THRESHOLD_BYTES = 10 * 1024 * 1024; // 10MB
+  private static final long QUEUE_FLUSH_THRESHOLD_BYTES = 200 * 1024 * 1024; // 200MB
```

Comment:
I think that this should be destination specific, the same way the optimal batch size is. It might be an issue when QUEUE_FLUSH_THRESHOLD_BYTES > getOptimalBatchSizeBytes

Reply:
done in https://github.com/airbytehq/airbyte/pull/32112/commits/fb7812476217283aabd463a5e50926cbaa526d9a